src/main/java/core/basesyntax/dataservices/DataConverterImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.dataservices;
+
+import core.basesyntax.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
```

Comment:
The class `DataConverterImpl` should be in its own package, such as `service.impl`, to follow the project structure guidelines.